syntax = "proto3";

package GptApp;

service GptMessage {

  rpc UpdateQueryResult (UpdateQueryResultRequest) returns (UpdateQueryResultReply) {}
  rpc UpdateDataFakerGenerateResult (UpdateDataFakerGenerateResultRequest) returns (UpdateDataFakerGenerateResultReply) {}

}

message UpdateQueryResultRequest {

  string reply_content = 1;
  string  query_message_id = 2;
  bool  interrupt = 3;
  string interrupt_reason = 4;
  bool error = 5;
  string error_code = 6;
  string error_detail = 7;

}

message UpdateQueryResultReply {

  bool success = 1;

}


message UpdateDataFakerGenerateResultRequest {
  string record_key=1;
  string file_path=2;
  string download_code=3;
}

message UpdateDataFakerGenerateResultReply {

  bool success = 1;

}
